server/utils/helpers.ts

```diff
@@ -81,3 +85,23 @@ export const prettifyErrorMessage = (rawErrorMessage: string) => {
     return `User ${match[2]} has no permissions to [${match[1]}].`;
   }
 };
+
+export function getClientBasedOnDataSource(
+  context: any,
```

Comment:
nit: this can be typed as `RequestHandlerContext`

Reply:
addressed in this commit - https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/719/commits/5057c0728b7ea9967312cfde0ad766f38e61dd85